pages/index.vue

```diff
@@ -1,23 +1,10 @@
 <template>
```

Comment:
Can you draw a tree or explain with `>` `<` relation marks how to think about `layouts/` vs. `pages/`? 

I see we have `default` and `home` layouts and I think I understand that those are the templates and then `pages/` are the actual views where you combine templates w/ data?

Reply:
layouts are different layouts of components, that can be set in pages -> pages use the default layout by default, but you can change the layout to some custom via this property

Reply:
That's good explanation in layman's terms thanks bax!